src/main/java/org/example/attorney/AttorneyScrapper.java

```diff
@@ -0,0 +1,20 @@
+package org.example.attorney;
+
+import org.jsoup.*;
+import org.jsoup.nodes.*;
+import org.jsoup.select.*;
+
+import java.io.*;
+import java.util.*;
+
+public class AttorneyScrapper {
+    public static List<String> getAttorneyLinks(String baseUrl) throws IOException {
+        Document page = Jsoup.connect(baseUrl).get();
+        List<String> attorneyLinks = new ArrayList<>();
+        Elements lawyersHtml = page.select("div[class=search-law__lawyer]");
+        for (Element attorney : lawyersHtml) {
+            attorneyLinks.add(attorney.selectFirst("a").attr("href"));
+        }
+        return attorneyLinks;
```

Comment:
where `PROFILE_URLS_SELECTOR = "div[class=search-law__lawyer] a"`
```suggestion
        return document.select(PROFILE_URLS_SELECTOR).stream()
        .map(e -> e.attr("href"))
        .toList();
```